setup.py

```diff
@@ -1,19 +1,164 @@
 import os
 import sys
+import platform
+import subprocess
+import git
+
+from distutils.core import setup, Extension 
 
 if os.name == 'nt':
     from distutils import msvc9compiler
     msvc9compiler.VERSION = 14.0 #Visual studio 2015
 
-if sys.version > '3':
-    setup_file = "setup_3.py"
+if sys.version <= '2':
```

Comment:
I prefer the code below when checking for values less than 3.
Do as you like.

if sys.version < '3':

if sys.version < '3':

Reply:
okay. i'll consider it. thank you